src/main/java/org/takes/rq/RqLive.java

```diff
@@ -84,12 +84,13 @@ private static Request parse(final InputStream input) throws IOException {
                 if (baos.size() == 0) {
                     break;
                 }
-                head.add(new String(baos.toByteArray()));
+                final String header = new String(baos.toByteArray());
```

Comment:
@lautarobock No need for this `header` variable, let's inline it with the method call below.